sonar-html-plugin/src/main/java/org/sonar/plugins/html/node/TagNode.java

```diff
@@ -49,8 +47,35 @@ public boolean equalsElementName(String elementName) {
     return StringUtils.equalsIgnoreCase(getLocalName(), elementName) || StringUtils.equalsIgnoreCase(getNodeName(), elementName);
   }
 
-  public String getAttribute(String attributeName) {
+  /**
+   *  This method takes into account the property binding mechanism of angular. See SONARHTML-92
+   */
+  @Nullable
+  public Attribute getProperty(String propertyName) {
+    String angularProperty = "[" + propertyName + "]";
+    for (Attribute a : attributes) {
```

Comment:
matter of style (so up to you to modify it or keep it as is) : 
`attributes.stream().map(Attribute::getName).filter(...).findFirst().orElse(null);`

Reply:
Ok. I'll keep it like this here then, to have consistent style inside the same file